Git-commit: e2155129599f96d29005721803ecb35bbf6d1ceb
From: Masahiro Yamada <masahiroy@kernel.org>
Date: Wed, 31 Mar 2021 22:38:02 +0900
Subject: [PATCH] kbuild: remove unneeded mkdir for external modules_install
References: bsn#19
Patch-mainline: v5.10.163

[ Upstream commit 4b97ec0e9cfd5995f41b9726c88566a31f4625cc ]

scripts/Makefile.modinst creates directories as needed.

Signed-off-by: Masahiro Yamada <masahiroy@kernel.org>
Stable-dep-of: c7b98de745cf ("phy: qcom-qmp-combo: fix runtime suspend")
Signed-off-by: Sasha Levin <sashal@kernel.org>
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Makefile b/Makefile
index 33422c7d149e..e1279904ca0e 100644
--- a/Makefile
+++ b/Makefile
@@ -1746,10 +1746,8 @@ $(MODORDER): descend
 PHONY += modules_install
 modules_install: _emodinst_ _emodinst_post
 
-install-dir := $(if $(INSTALL_MOD_DIR),$(INSTALL_MOD_DIR),extra)
 PHONY += _emodinst_
 _emodinst_:
-	$(Q)mkdir -p $(MODLIB)/$(install-dir)
 	$(Q)$(MAKE) -f $(srctree)/scripts/Makefile.modinst
 
 PHONY += _emodinst_post
-- 
2.35.3

